decred/decred/dcr/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+Copyright (c) 2019-2020, the Decred developers
+See LICENSE for details
+"""
+
+from urllib.parse import urlsplit, urlunsplit
+
+
+def getSocketURLs(url):
```

Comment:
Moved here from `dcrdata.py` to be also used by `rpc.py` in place of `websocketURI`.